testing_api/rest.go

```diff
@@ -0,0 +1,97 @@
+package rest
+
+import (
+	"fmt"
+
+	"github.com/go-resty/resty/v2"
+)
+
+// SuccessRequest for ,if request success then to get token
+type SuccessRequest struct {
+	Token     string `json:"jwt"`
+	AccountID string `json:"accountId"`
+}
+
+// FailedRequest fail return error
+type FailedRequest struct {
+	Msg string `json:"message"`
+}
+
+// UserData contains the details of credentials
+type UserData struct {
```

Comment:
Are we modelling some specific things?
Is it KuberaUser or DirectorUser or something else?
Let us try to put specific names for structures, variables & so on